src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient/Models/Arkivmelding.cs

```diff
@@ -0,0 +1,399 @@
+        public int AntallFiler { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Mappe
+        /// </summary>
+        [XmlElement(ElementName = "mappe")]
+        public List<Mappe> Mappe { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the MeldingId
+        /// </summary>
+        [XmlElement(ElementName = "meldingId", Namespace = "http://www.arkivverket.no/standarder/noark5/arkivmelding")]
+        public string MeldingId { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the SchemaLocation
+        /// </summary>
+        [XmlAttribute(AttributeName = "schemaLocation", Namespace = "http://www.w3.org/2001/XMLSchema-instance")]
+        public string SchemaLocation { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the System
+        /// </summary>
+        [XmlElement(ElementName = "system", Namespace = "http://www.arkivverket.no/standarder/noark5/arkivmelding")]
+        public string System { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Tidspunkt
+        /// </summary>
+        [XmlElement(ElementName = "tidspunkt", Namespace = "http://www.arkivverket.no/standarder/noark5/arkivmelding")]
+        public string Tidspunkt { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Xmlns
+        /// </summary>
+        [XmlAttribute(AttributeName = "xmlns", Namespace = "http://www.arkivverket.no/standarder/noark5/arkivmelding")]
+        public string Xmlns { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Xsi
+        /// </summary>
+        [XmlAttribute(AttributeName = "xsi", Namespace = "http://www.w3.org/2000/xmlns/")]
+        public string Xsi { get; set; }
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Dokumentobjekt"/> class.
+    /// </summary>
+    [XmlRoot(ElementName = "dokumentobjekt")]
+    public class Dokumentobjekt
+    {
+        /// <summary>
+        ///  Gets or sets the Versjonsnummer
+        /// </summary>
+        [XmlElement(ElementName = "versjonsnummer")]
+        public int Versjonsnummer { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Variantformat
+        /// </summary>
+        [XmlElement(ElementName = "variantformat")]
+        public string Variantformat { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the OpprettetDato
+        /// </summary>
+        [XmlElement(ElementName = "opprettetDato")]
+        public DateTime OpprettetDato { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the OpprettetAv
+        /// </summary>
+        [XmlElement(ElementName = "opprettetAv")]
+        public string OpprettetAv { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the ReferanseDokumentfil
+        /// </summary>
+        [XmlElement(ElementName = "referanseDokumentfil")]
+        public List<string> ReferanseDokumentfil { get; set; }
```

Comment:
based on the testdata this seems like a string and not a list? can it hold multiple file names?